application/web-app/react-client/src/components/CircularProgress.js

```diff
@@ -27,7 +27,12 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-export default function CircularIndeterminate() {
```

Comment:
👍 , as discussed this is because documentation generation doesn't work from the direct export syntax